mne/io/ctf/tests/test_ctf.py

```diff
@@ -211,4 +211,11 @@ def test_read_spm_ctf():
     assert_equal(extras['n_samp'], raw.n_times)
     assert_false(extras['n_samp'] == extras['n_samp_tot'])
 
+    # Test that LPA, nasion and RPA are correct.
+    cardinals = {d['ident']: d['r'] for d in raw.info['dig']}
+    assert_true(cardinals[1][0] < cardinals[2][0] < cardinals[3][0])  # x coord
+    assert_true(cardinals[1][1] < cardinals[2][1])  # y coord
+    assert_true(cardinals[3][1] < cardinals[2][1])  # y coord
```

Comment:
Well actually it's in `COORD_UNKNOWN` coordinates, so this check isn't actually valid yet. Once the points are in Neuromag head coordinate frame it would be fine.